frontend/src/metabase/ui/components/inputs/Input/Input.styled.tsx

```diff
@@ -73,9 +73,6 @@ export const getInputOverrides = (): MantineThemeOverride["components"] => ({
           "&:focus": {
             borderColor: theme.colors.brand[1],
           },
-          "&:read-only:not(:disabled)": {
```

Comment:
Select is always `readOnly` on `Input` level. We should either move this to `TextInput` and `NumberInput` or drop entirely. Currently nothing is exposed in `Select` style API to be able to do the same for `Select`, e.g. to add a special border color for read-only but not disabled inputs.